File: Data Structure/Lab 1/AreaOfCircle.c
//WAP to find area of a circle

#include<stdio.h>
#define pie 3.14
void main(){
    float r;
    float area = 0;

    printf("Enter radius of a circle");
    scanf("%f",&r);

    area = pie * r * r; 

    printf("Area of circle is : %.2f",area);
}

File: Data Structure(Java)/Lab 1/AreaofCircle.java
//WAP to calculate area of a Circle (A = πr2). 

import java.util.Scanner;

public class AreaofCircle{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter radius:");
        int r = sc.nextInt();

        double area = Math.PI * r * r;

        System.out.println(area);

        sc.close();
    }
}